### DISCUSSION TITLE
I used rsbuild to open a new project and configured unocss according to the documentation, but why did it report an error?

### DISCUSSION THREAD

**Discussion by judapeng** [web-infra-dev/rsbuild#3314] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/5342837b-2d80-4b9f-9107-f1d919834f20)
![image](https://github.com/user-attachments/assets/587fc4f3-5975-422e-abde-a39527dc7ebe)
![image](https://github.com/user-attachments/assets/1009dc4d-39ee-4c2f-b346-8b19b739bc9b)

**Comment by chenjiahan** [comment, marked as answer]
Are you referring to the "Unknown at rule" error? This is an IDE prompt and has nothing to do with Rsbuild.
